configs/benchmark-rongjian.json

```diff
@@ -0,0 +1,101 @@
+{
+  "description": "aws 464/600 nodes on 4 shard in 4 region, reserve 34 nodes for FN",
+  "client": {
+    "num_vm": 0,
```

Comment:
This is if you want to launch your own devnet? You just want launch your explorer nodes right.